db/src/main/kotlin/edu/artic/db/ApiModule.kt

```diff
@@ -0,0 +1,23 @@
+package edu.artic.db
+
+import dagger.Module
+import dagger.Provides
+import retrofit2.Retrofit
+import javax.inject.Named
+import javax.inject.Singleton
+
+@Module
+abstract class ApiModule {
+
+    @Module
+    companion object {
+        @JvmStatic
+        @Provides
+        @Singleton
+        @Named("BlobRetrofit")
```

Comment:
Please rename this. BlobRetrofit is weird. and make a constant for the Named injection.